lib/cloud.proxy.js

```diff
@@ -83,6 +88,47 @@ IoTKitCloud.prototype.isActivated = function () {
     }
     return false;
 };
+
+IoTKitCloud.prototype.ensureFreshness = function (callback) {
```

Comment:
Can you change the name of this function to something more specific like "checkDeviceToken".

"ensureFreshness" looks and sounds weird to me although I understand where you were coming from :)